phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ApplicationSideBar.java

```diff
@@ -158,7 +158,7 @@ private ToggleButton createAllCategoriesToggleButton() {
     private ToggleButton createCategoryToggleButton(CategoryDTO category) {
         final LeftToggleButton categoryButton = new LeftToggleButton(category.getName());
 
-        categoryButton.setId(String.format("%sButton", category.getName().toLowerCase()));
+        categoryButton.setId(String.format("%sButton", category.getId().toLowerCase()));
```

Comment:
The problem I see here is with category ids such as `arcade games`.
The created button id string for such category ids would look like this: `#arcade gamesButton`.
This id string would be invalid, because it contains a space character and is therefore interpreted as a tag with the id `arcade`, which contains another element `gamesButton`.